utils/crypto/bls/ciphersuite.go

```diff
@@ -0,0 +1,12 @@
+// Copyright (C) 2019-2024, Ava Labs, Inc. All rights reserved.
+// See the file LICENSE for licensing terms.
+
+package bls
+
+var (
+	// The ciphersuite is more commonly known as G2ProofOfPossession.
+	// There are two digests to ensure that message space for normal
+	// signatures and the proof of possession are distinct.
+	CiphersuiteSignature         = []byte("BLS_SIG_BLS12381G2_XMD:SHA-256_SSWU_RO_POP_")
+	CiphersuiteProofOfPossession = []byte("BLS_POP_BLS12381G2_XMD:SHA-256_SSWU_RO_POP_")
+)
```

Comment:
Is there any way to make these `const`s if we are exporting them?
```suggestion
const (
	// The ciphersuite is more commonly known as G2ProofOfPossession.
	// There are two digests to ensure that message space for normal
	// signatures and the proof of possession are distinct.
	CiphersuiteSignature         = "BLS_SIG_BLS12381G2_XMD:SHA-256_SSWU_RO_POP_"
	CiphersuiteProofOfPossession = "BLS_POP_BLS12381G2_XMD:SHA-256_SSWU_RO_POP_"
)
```
I think it was done as a var because I was concerned that a `[]byte("")` cast would perform memory allocations... But I suppose we could use unexported vars for that as well.